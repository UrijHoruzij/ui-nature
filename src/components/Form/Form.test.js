import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { Form } from "../";
import ThemeProvider from "../../ThemeProvide";

describe("Form component", () => {
  test("Render form", () => {
    const component = (
      <ThemeProvider>
        <Form></Form>
      </ThemeProvider>
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
