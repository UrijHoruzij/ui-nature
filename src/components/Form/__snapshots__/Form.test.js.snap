// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component Render form 1`] = `
<form
  className=""
/>
`;
